File: app/src/main/java/org/mifos/mobile/ui/views/LoanAccountWithdrawView.java
package org.mifos.mobile.ui.views;

import org.mifos.mobile.ui.views.base.MVPView;

/**
 * Created by dilpreet on 7/6/17.
 */

public interface LoanAccountWithdrawView extends MVPView {

    public void showLoanAccountWithdrawSuccess();

    public void showLoanAccountWithdrawError(String message);
}

File: app/src/main/java/org/mifos/mobile/ui/views/LoanAccountWithdrawView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.ui.views.base.MVPView

/**
 * Created by dilpreet on 7/6/17.
 */
interface LoanAccountWithdrawView : MVPView {
    fun showLoanAccountWithdrawSuccess()
    fun showLoanAccountWithdrawError(message: String?)
}